internal/gengapic/client_init.go

```diff
@@ -310,3 +311,22 @@ func generateDefaultMTLSEndpoint(defaultEndpoint string) string {
 	}
 	return defaultEndpoint
 }
+
+// generateDefaultAudience transforms a host into a an audience that can be used
+// as the `aud` claim in a JWT.
+func generateDefaultAudience(host string) string {
```

Comment:
Are we guaranteed that the host is a proper host (ie doesn't have a path)? I'd be inclined to defensively remove it (and include that in tests).

Reply:
Guaranteed, no. But it does not happen today I can fairly certainly say. On L63 of this file we append `:443` to the the end of the host. If there were path fragments this should be an invalid host and the client would not be able to talk to the backend.